src/utils/testing/helpers.py

```diff
@@ -30,7 +30,7 @@
 from cms import models as cms_models
 
 
-def create_user(username, roles=None, journal=None, **attrs):
+def create_user(username, roles=None, journal=None, is_staff=False, **attrs):
```

Comment:
I think this should work as part of `**attrs` unless I'm missing something

Reply:
As mentioned in the message above, it doesn't.

Reply:
I'll double check this in a mo.